components/modal/modal.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Aamir Nathani" />
+    <title>Modal - NOMARK UI</title>
+    <link rel="icon" href="../../assests/icon-logo.png">
+
+    <!-- External Imports -->
+    <link rel="stylesheet" href="../button/button.css">
+    <link rel="stylesheet" href="./modal.css">
+    <link rel="stylesheet" href="../../styles/main.css">
+
+</head>
+
+<body>
+    <div class="section-center">
+        <!-- navbar -->
+        <nav class="navbar flex items-center justify-between pt-1 section-center z-5">
+            <a class="humburger-icon icon-open font-size-6"><i class="fas fa-bars ml-5"></i></a>
+            <a class="humburger-icon icon-close font-size-6 displayNone"><i class="fas fa-bars ml-5"></i></a>
+            <a href="../../index.html" class="mr-5"><img src="../../assests/logo-primary.png" alt="NOMARK"
+                    class="nomark-logo"></a>
+        </nav>
+        <!-- side-bar -->
+        <div class="sidebar sidebarRemove flex flex-col scroll-style z-4 pr-7 p-2">
+            <h3 class="sidebar-heading font-size-6">Getting Started</h3>
+            <ul>
+                <li><a href="../../documentation/doc.html" class="sidebar-items">Installation</a></li>
+            </ul>
+            <h3 class="sidebar-heading font-size-6">Components</h3>
+            <ul>
+                <li><a href="../avatar/avatar.html" class="sidebar-items">Avatar</a></li>
+                <li><a href="../alert/alert.html" class="sidebar-items">Alert</a></li>
+                <li><a href="../badge/badge.html" class="sidebar-items">Badge</a></li>
+                <li><a href="../button/button.html" class="sidebar-items">Button</a></li>
+                <li><a href="../card/card.html" class="sidebar-items">Card</a></li>
+                <li><a href="../image/image.html" class="sidebar-items">Image</a></li>
+                <li><a href="../input/input.html" class="sidebar-items">Input</a></li>
+                <li><a href="../typography/typography.html" class="sidebar-items">Typography</a></li>
+                <li><a href="../list/list.html" class="sidebar-items">Lists</a></li>
+                <li><a href="../navbar/navbar.html" class="sidebar-items">Navigation</a></li>
+                <li><a href="./model.html" class="sidebar-items active">Modal</a></li>
```

Comment:
plz, check the name once here href is "model.html"

Reply:
fix the file path